docs/translations/api-docs/dashboard-layout/dashboard-layout.json

```diff
@@ -1,5 +1,10 @@
 {
   "componentDescription": "",
-  "propDescriptions": { "children": { "description": "The content of the dashboard." } },
+  "propDescriptions": {
+    "children": { "description": "The content of the dashboard." },
+    "initialNavigationOpen": {
+      "description": "Whether the mobile navigation drawer should start open."
```

Comment:
Should this be external API? Why would users need this?
Besides, can't we, for demo purposes just force the large screen behavior. Maybe change the breakpoints or something. But in a hidden way, users shouldn't copy+paste our hacks accidentally.

Reply:
It should be internal API if possible, just not sure what the best way to do that is? Will try to find a solution.

Also wasn't sure if it's better to have the mobile or the desktop version on the docs, so I guess we can try to use the desktop one there or change the breakpoints.